qp_klp/__init__.py

```diff
@@ -13,7 +13,7 @@ class QiitaPluginAdmin(QiitaPlugin):
     _plugin_type = "private"
 
 
-__version__ = '2023.05'
+__version__ = '2024.11'
```

Comment:
Not sure I follow - the version number has been unchanged through several of the last releases. I updated it to 2024.11 for the upcoming release. However since it's now 2024.12, I can update it again. Does that sound all right?

Reply:
Release https://github.com/qiita-spots/qp-knight-lab-processing/releases/tag/2024.09 cut for previous release.